vms/platformvm/txs/builder/context.go

```diff
@@ -1,32 +0,0 @@
-// Copyright (C) 2019-2024, Ava Labs, Inc. All rights reserved.
-// See the file LICENSE for licensing terms.
-
-package builder
-
-import (
-	"time"
-
-	"github.com/ava-labs/avalanchego/snow"
-	"github.com/ava-labs/avalanchego/vms/platformvm/config"
-
-	walletbuilder "github.com/ava-labs/avalanchego/wallet/chain/p/builder"
-)
-
-func NewContext(
```

Comment:
as mentioned in the wallet file, I'd keep it here and it is only used in the txbuilder and unfortunately in a couple of UTs, which force us to export this context